### ISSUE TITLE
No such file or directory: '.git/COMMIT_EDITMSG'

### ISSUE THREAD

**Issue by shenxianpeng** [commit-check/commit-check#88]
### describe your issue

```bash
check commit message.....................................................Failed
- hook id: check-message
- exit code: 3

An unexpected error has occurred: FileNotFoundError: [Errno 2] No such file or directory: '.git/COMMIT_EDITMSG'
Check the log at /tmp/cache/commit-check/commit-check.log
```

More details: https://results.pre-commit.ci/run/github/485387677/1681156478.A-v-8Se8SZGUgEcJW7JhLQ

### commit-check version

commit-check v0.5.4

### .commit-check.yml

```yaml
*
```
